.github/workflows/build.yml

```diff
@@ -10,7 +10,7 @@ jobs:
     strategy:
       fail-fast: false
       matrix:
-        version: [3.7, 3.8]
+        version: [3.8]
```

Comment:
Any reason we need to support 3.7? It seems latest versions of major scientific packages like scipy, numpy, matplotlib do not support it any more.